src/PrestaShopBundle/Translation/Api/InternationalApi.php

```diff
@@ -36,7 +36,10 @@ public function getTranslations()
         return array(
             'button_reset' => $this->translator->trans('Reset', array(), 'Admin.Actions'),
             'button_save' => $this->translator->trans('Save', array(), 'Admin.Global'),
+            'button_leave' => $this->translator->trans('Leave Anyway', array(), 'Admin.Global'),
```

Comment:
Could we have "Leave anyway" (no capital A) and domain "Admin.Notifications.Warning" ?